.gitignore

```diff
@@ -42,41 +70,34 @@ node_modules/
 lifttemplate.iml
 lifttemplate.ipr
 lifttemplate.iws
-src/main/scala/rebel.xml
 *.obj
 *.zip
 *.skp
 *.mtl
-public/bundle
-public/test-bundle
-public-test/test-bundle
-public/docs
-public/commit.txt
 /knowledge
 conf/github.conf
 tmp/
 errorShots/
 flow-coverage/
-public/docs/frontend-api
-dev-db/
-docker-compose.override.yml
-frontend/javascripts/test/snapshots/debug-htmls/*
-!frontend/javascripts/test/snapshots/debug-htmls/.gitkeep
-frontend/javascripts/test/snapshots/type-check/*
-!frontend/javascripts/test/snapshots/type-check/.gitkeep
-.eslintcache
+
+
+# Sublime 
 *.sublime-project
 *.sublime-workspace
-**/screenshots/*.diff.png
-**/screenshots/*.new.png
+
+# Testing
 .nyc_output
 coverage
 coverage-ts
-*.vscode
-.bsp
-temp-webknossos-schema**
-conf/application.conf-e
-.env
-.bloop
-.metals
-metals.sbt
+**/screenshots/*.diff.png
+**/screenshots/*.new.png
+
+# Yarn 
```

Comment:
Added a new `yarn` group as described in their docs: https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored